tests/query/v1/test_fetch_vertex.py

```diff
@@ -335,6 +335,144 @@ def test_fetch_vertex_get_all_with_yield_invalid(self):
         resp = self.execute_query(query)
         self.check_resp_failed(resp)
 
+    def test_fetch_vertex_on_multi_tags(self):
+        query = 'FETCH PROP ON player, team "Boris Diaw"'
+        resp = self.execute_query(query)
+        expect_column_names = ['VertexID', 'player.name', 'player.age', 'team.name']
+        expect_result = [['Boris Diaw', 'Boris Diaw', 36, T_EMPTY]]
+        self.check_resp_succeeded(resp)
+        self.check_column_names(resp, expect_column_names)
+        self.check_out_of_order_result(resp, expect_result)
+
+        # multi vertices
+        query = 'FETCH PROP ON player, bachelor "Tim Duncan", "Boris Diaw"'
+        resp = self.execute_query(query)
+        expect_column_names = ['_vid', 'player.name', 'player.age', 'bachelor.name', 'bachelor.speciality']
+        expect_result = [['Tim Duncan', 'Tim Duncan', 42, "Tim Duncan", "psychology"],
+                         ['Boris Diaw', 'Boris Diaw', 36, T_EMPTY, T_EMPTY]]
+        self.check_resp_succeeded(resp)
+        self.check_out_of_order_result(resp, expect_result)
+
+        # from input
+        query = '''GO FROM "Boris Diaw" over like YIELD like._dst as id
+            | FETCH PROP ON player, bachelor $-.id'''
+        resp = self.execute_query(query)
+        expect_column_names = ['_vid', 'player.name', 'player.age', 'bachelor.name', 'bachelor.speciality']
```

Comment:
why the column name is different between line 360 and line 377, `_vid` vs `VertexID` ?

Reply:
Good catch. It's a mistake. Had fixed.